### ISSUE TITLE
[Unblock Legitimate Site] Urgent: Critical Issue on BlockDAG Platform - Immediate Action Required

### ISSUE THREAD

**Issue by BlockDAGNetwork** [MetaMask/eth-phishing-detect#28233]
Dear BlockAid/Metamask Team,

I hope this message finds you well. We're urgently reaching out from BlockDAG regarding a pressing issue that demands your immediate attention.

As previously communicated, our platform, https://blockdag.network/, is currently facing a significant obstacle that directly impacts user experience and trust.

Despite our prior outreach, the troubling message "THIS IS A DECEPTIVE REQUEST" by BlockAid continues to persist when users attempt to purchase coins via ConnectWallet, specifically when connecting through MetaMask.

This persistent prompt not only confuses our users but also undermines the trust and credibility we've worked tirelessly to establish. We're already experiencing a surge in negative feedback and a concerning decline in user confidence due to this issue.

To safeguard our reputation and ensure uninterrupted operations, we implore you to take immediate action to remove this deceptive request message from our platform. This action is critical to restoring trust and confidence in our service.

Please note that while users can still access our site, this recurring message severely disrupts their experience, particularly during crucial transactions.

We understand the complexity of resolving such issues and appreciate your prompt attention to this matter. Your swift response and resolution are indispensable in mitigating further damage to our platform's reputation.

For your convenience, here are our official communication channels and validation tool:

Website: https://blockdag.network/
Telegram: https://t.me/blockDAGnetworkOfficial
Instagram: https://www.instagram.com/blockdagnetwork/
Twitter/X: https://twitter.com/blockdagnetwork (Verified)
Validation Tool: https://blockdag.network/validation

We are an official presale, and it's imperative that any blocks or red alert messages associated with our platform be promptly addressed and removed.

Additionally, I've included some official articles about our platform for your reference:

"(https://www.analyticsinsight.net/blockdag-networks-10000x-profits-potential-amid-pavel-durovs-new-strategy-for-toncoin-trading-and-optimism-surge/)"
"(https://www.ndtv.com/partner-content/crypto-watchlist-matic-coin-and-retik-crypto-finance-lose-to-blockdag-networks-2-million-mega-giveaway-5180854)"
"(https://zycrypto.com/blockdag-network-rise-on-top-of-crypto-watchlist-sparking-a-surge-amid-btcmtx-presale/)"
"(https://cryptopotato.com/blockdag-raises-1-million-in-presale-and-announces-2-million-giveaway/)"
![Metamask_BlockDAG_deceptiveRequest_desktop](https://github.com/MetaMask/eth-phishing-detect/assets/161131059/86e77839-64fd-4987-be9c-d7c7d09442dd)

![Metamask_BlockDAG_deceptiveRequest_desktop](https://github.com/MetaMask/eth-phishing-detect/assets/161131059/4eef5c12-c7c7-44e2-b320-d2cfa77948ef)

Your cooperation and understanding of the urgency of this situation are greatly appreciated. We eagerly await your prompt resolution to restore trust and confidence in our platform.

Thank you for your attention to this matter.

Warm regards,
- [x] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by AlexHerman1** [comment]
Hi,
There is another issue about this open already. did you use the reporting-form in-app to report the false positive?

**Comment by bschorchit** [comment]
For these cases, clicking on `Report an issue` from within the warning is the best way to report it and get a timely review. 
![Screenshot 2024-03-05 at 13 14 54](https://github.com/MetaMask/eth-phishing-detect/assets/34968356/bdb7eea4-5b1b-452a-a804-69bad6155a24)

**Comment by AlexHerman1** [comment]
@BlockDAGNetwork can you confirm this has been resolved?

**Comment by AlexHerman1** [comment]
Glad to hear this, closing this issue.

**Comment by frovech** [comment]
I'm concerned there's a possible rug pull in progress.
 
The rug pull project is known as BlockDAG, located at https://blockdag.network/
 
I'm going to give you the details, so you can get your team to monitor these addresses and stop the perpetrator. 
 
The deposit address for BlockDAG is https://bscscan.com/address/0x430f4919d679b02dca4572579caabb7676d65f1e.
 
Funds from the same address  0x430f4919d679b02dca4572579caabb7676d65f1e on Ethereum are converted to USDT at the address 0x1A8993bb740D91498dFa0F7fe5F16Cc0d4C687Dc.
 
This Ethereum address: 0x1A8993bb740D91498dFa0F7fe5F16Cc0d4C687Dc then sends all the USDT to Binance 14.
 
Why I'm concerned it's a rug pull is because the BSC address 0x430f4919d679b02dca4572579caabb7676d65f1e is using the project's fund to create meme tokens, which is outside the remit of the project address.
 
In addition, the project was started sometime in late 2023, but it only created its BDAG contract address today. Also, today, it created a Uniswap token which currently is trading lower than the presale price of the project.
https://app.uniswap.org/explore/tokens/ethereum/0xa5d6abc273c114ef7071f86c834ee14ea63a1948
 
Today, when a couple of us started asking questions, then the project token, BDAG, was created on Ethereum: https://etherscan.io/address/0xa5d6abc273c114ef7071f86c834ee14ea63a1948

In addition to not answering our questions, we were kicked out and silenced on the social media channels.
 
Can you please ask your team to monitor the relevant Ethereum and BSC address listed in this message, and prevent massive amounts of currency from being moved out, as well as reinstate the warnings, that it is a possible rug-pull, and for users of Metamask to be careful?
 
Thank you.

**Comment by BlockDAGNetwork** [comment]
Hello, 

We have been receiving multiple DECEPTIVE and error messages when our users connect their wallets to our site.

We have already provided sufficient information to prove that we are a legitimate site and the presale is for raising funds in order to build the technology for blockchain and also the marketing.

I would like yo also clarify that the addresses are made by imposters and not us.

These address below have no connection to our project.

1. [](https://etherscan.io/address/0xa5d6abc273c114ef7071f86c834ee14ea63a1948)

2. [](https://app.uniswap.org/explore/tokens/ethereum/0xa5d6abc273c114ef7071f86c834ee14ea63a1948)

I have added some attachments of examples of error messages which our users are receiving.
![image (7)](https://github.com/MetaMask/eth-phishing-detect/assets/161131059/c7863e41-3b8c-4d02-89f5-ff1e498f6115)
![image (6)](https://github.com/MetaMask/eth-phishing-detect/assets/161131059/c1874aa8-dc82-4e7f-ba28-e8ec1900cdda)
![IMG_4448](https://github.com/MetaMask/eth-phishing-detect/assets/161131059/a410ad49-07ef-4008-82d7-8cc5a431ae41)


Please remove these these as high priority as this is causing sever negative effects for Our project.

We hope to have a prompt response to this request for this issue to be resolved immediately.

Thanks, BlockDAG Team.
